### PR TITLE
gst-editing-services 1.8.0 (new formula)

### PR DESCRIPTION
**Pull request by aleb**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Does your submission pass `brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Have you built your formula locally prior to submission with `brew install <formula>`?

### CODE REVIEW

File: Formula/gst-editing-services.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      --disable-docbook
+    ]
+
+    system "./configure", *args
+    system "make"
+    system "make", "install"
+  end
+
+  test do
+    system "ges-launch-1.0", "--ges-version"
```
Comment (mikemcquaid) [inline Formula/gst-editing-services.rb]: Use `system "#{bin}/ges-launch-1.0 ..` to set the full path.

File: Formula/gst-editing-services.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  depends_on "gst-plugins-base"
+
+  def install
+    args = %W[
+      --prefix=#{prefix}
+      --disable-gtk-doc
+      --disable-docbook
+    ]
+
+    system "./configure", *args
```
Comment (mikemcquaid) [inline Formula/gst-editing-services.rb]: You don't have to use `args` here as there is no variable args.

File: Formula/gst-editing-services.rb (hunk 1 above)
Comment (aleb) [inline Formula/gst-editing-services.rb]: Done

File: Formula/gst-editing-services.rb (hunk 2 above)
Comment (aleb) [inline Formula/gst-editing-services.rb]: Done

**Comment by DomT4** [comment]
Seems reasonable to me.

Merged in https://github.com/Homebrew/homebrew-core/commit/320a3698314b9b95d2c5c0561a5e77c47fcd6083. Thank you for your contribution to Homebrew @aleb; we appreciate it! :smiley_cat:

**Pull request closed without merging**